spec/per_seat_server_spec.rb

```diff
@@ -0,0 +1,145 @@
+RSpec.describe "full integration path" do
+  it "just works" do
+    # Get the index html page
+    response = get("/")
+    expect(response).not_to be_nil
+
+    # Get the config
+    config, _ = get_json("/config")
+    expect(config.keys).to contain_exactly("publishableKey")
+
+    # Create the customer
+    customer, _ = post_json("/create-customer", {
+      email: "jenny.rosen@example.com",
+    })
+    expect(customer["customer"]["id"]).to start_with("cus_")
+    customer_id = customer["customer"]["id"]
+
+    # Create the subscription
+    subscription, _ = post_json("/create-subscription", {
+      customerId: customer_id,
+      paymentMethodId: "pm_card_visa",
+      priceId: "BASIC",
+      quantity: 3,
+    })
+    expect(subscription).not_to be_nil
+    expect(subscription["id"]).to start_with("sub_")
```

Comment:
should we check that the quantity was set correctly?

Reply:
great call! added